website/content/docs/using-netlify-cms-with-gatsby.md

```diff
@@ -0,0 +1,120 @@
+# Using Netlify CMS with Gatsby
```

Comment:
In order for this page to appear on the docs site, you'll need to move this title into a front matter section with `title`, `weight`, and `group`. 

I think `guides` probably makes most sense for the `group` (agree, @tomrutgers @erquhart ?)

For the weight, I'd pick a lower number than [custom-widgets](https://github.com/netlify/netlify-cms/blob/master/website/content/docs/custom-widgets.md), which is 35. Maybe 20? That would leave a fair amount of room on either side.